### PR TITLE
r/s3_bucket: read-only `acceleration_status` argument

### PR DESCRIPTION
**Pull request by anGie44**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates #4418
Relates #20433

Output from acceptance testing:

```
--- PASS: TestAccS3BucketAnalyticsConfiguration_WithFilter_empty (15.92s)
--- PASS: TestAccS3BucketAnalyticsConfiguration_WithFilter_multipleTags (170.79s)
--- PASS: TestAccS3BucketAnalyticsConfiguration_WithFilter_prefix (173.46s)
--- PASS: TestAccS3BucketAnalyticsConfiguration_WithFilter_prefixAndTags (174.41s)
--- PASS: TestAccS3BucketAnalyticsConfiguration_WithFilter_remove (173.31s)
--- PASS: TestAccS3BucketAnalyticsConfiguration_WithFilter_singleTag (173.51s)
--- PASS: TestAccS3BucketAnalyticsConfiguration_WithStorageClassAnalysis_default (103.19s)
--- PASS: TestAccS3BucketAnalyticsConfiguration_WithStorageClassAnalysis_empty (15.04s)
--- PASS: TestAccS3BucketAnalyticsConfiguration_WithStorageClassAnalysis_full (102.43s)
--- PASS: TestAccS3BucketAnalyticsConfiguration_basic (103.17s)
--- PASS: TestAccS3BucketAnalyticsConfiguration_removed (158.84s)
--- PASS: TestAccS3BucketAnalyticsConfiguration_updateBasic (250.25s)

--- PASS: TestAccS3BucketDataSource_basic (93.30s)
--- PASS: TestAccS3BucketDataSource_website (92.71s)

--- PASS: TestAccS3BucketIntelligentTieringConfiguration_Filter (388.23s)
--- PASS: TestAccS3BucketIntelligentTieringConfiguration_basic (104.63s)
--- PASS: TestAccS3BucketIntelligentTieringConfiguration_disappears (92.88s)

--- PASS: TestAccS3BucketInventory_basic (105.09s)
--- PASS: TestAccS3BucketInventory_encryptWithSSEKMS (105.51s)
--- PASS: TestAccS3BucketInventory_encryptWithSSES3 (104.33s)

--- PASS: TestAccS3BucketMetric_basic (102.63s)
--- PASS: TestAccS3BucketMetric_withEmptyFilter (16.29s)
--- PASS: TestAccS3BucketMetric_withFilterMultipleTags (172.37s)
--- PASS: TestAccS3BucketMetric_withFilterPrefix (174.48s)
--- PASS: TestAccS3BucketMetric_withFilterPrefixAndMultipleTags (174.31s)
--- PASS: TestAccS3BucketMetric_withFilterPrefixAndSingleTag (172.99s)
--- PASS: TestAccS3BucketMetric_withFilterSingleTag (171.45s)

--- PASS: TestAccS3BucketNotification_LambdaFunctionLambdaFunctionARN_alias (120.11s)
--- PASS: TestAccS3BucketNotification_Topic_multiple (104.80s)
--- PASS: TestAccS3BucketNotification_lambdaFunction (129.65s)
--- PASS: TestAccS3BucketNotification_queue (132.34s)
--- PASS: TestAccS3BucketNotification_topic (102.56s)
--- PASS: TestAccS3BucketNotification_update (188.99s)

--- PASS: TestAccS3BucketObjectDataSource_allParams (91.84s)
--- PASS: TestAccS3BucketObjectDataSource_basic (87.66s)
--- PASS: TestAccS3BucketObjectDataSource_basicViaAccessPoint (94.17s)
--- PASS: TestAccS3BucketObjectDataSource_bucketKeyEnabled (91.41s)
--- PASS: TestAccS3BucketObjectDataSource_kmsEncrypted (91.21s)
--- PASS: TestAccS3BucketObjectDataSource_leadingSlash (162.88s)
--- PASS: TestAccS3BucketObjectDataSource_multipleSlashes (163.53s)
--- PASS: TestAccS3BucketObjectDataSource_objectLockLegalHoldOff (92.44s)
--- PASS: TestAccS3BucketObjectDataSource_objectLockLegalHoldOn (88.20s)
--- PASS: TestAccS3BucketObjectDataSource_readableBody (92.39s)
--- PASS: TestAccS3BucketObjectDataSource_singleSlashAsKey (92.30s)

--- PASS: TestAccS3BucketObject_acl (255.57s)
--- PASS: TestAccS3BucketObject_bucketBucketKeyEnabled (94.12s)
--- PASS: TestAccS3BucketObject_content (103.96s)
--- PASS: TestAccS3BucketObject_contentBase64 (89.63s)
--- PASS: TestAccS3BucketObject_defaultBucketSSE (93.29s)
--- PASS: TestAccS3BucketObject_empty (104.30s)
--- PASS: TestAccS3BucketObject_etagEncryption (102.77s)
--- PASS: TestAccS3BucketObject_ignoreTags (170.08s)
--- PASS: TestAccS3BucketObject_kms (102.29s)
--- PASS: TestAccS3BucketObject_metadata (257.10s)
--- PASS: TestAccS3BucketObject_noNameNoKey (23.41s)
--- PASS: TestAccS3BucketObject_nonVersioned (100.87s)
--- PASS: TestAccS3BucketObject_objectBucketKeyEnabled (95.84s)
--- PASS: TestAccS3BucketObject_objectLockLegalHoldStartWithNone (244.29s)
--- PASS: TestAccS3BucketObject_objectLockLegalHoldStartWithOn (170.47s)
--- PASS: TestAccS3BucketObject_objectLockRetentionStartWithNone (242.77s)
--- PASS: TestAccS3BucketObject_objectLockRetentionStartWithSet (317.22s)
--- PASS: TestAccS3BucketObject_source (104.04s)
--- PASS: TestAccS3BucketObject_sourceHashTrigger (177.44s)
--- PASS: TestAccS3BucketObject_sse (105.50s)
--- PASS: TestAccS3BucketObject_storageClass (406.67s)
--- PASS: TestAccS3BucketObject_tags (328.40s)
--- PASS: TestAccS3BucketObject_tagsLeadingMultipleSlashes (319.54s)
--- PASS: TestAccS3BucketObject_tagsLeadingSingleSlash (325.90s)
--- PASS: TestAccS3BucketObject_tagsMultipleSlashes (316.99s)
--- PASS: TestAccS3BucketObject_updateSameFile (165.69s)
--- PASS: TestAccS3BucketObject_updates (178.30s)
--- PASS: TestAccS3BucketObject_updatesWithVersioning (175.50s)
--- PASS: TestAccS3BucketObject_updatesWithVersioningViaAccessPoint (167.00s)
--- PASS: TestAccS3BucketObject_withContentCharacteristics (90.98s)

--- PASS: TestAccS3BucketObjectsDataSource_all (171.09s)
--- PASS: TestAccS3BucketObjectsDataSource_basic (170.17s)
--- PASS: TestAccS3BucketObjectsDataSource_basicViaAccessPoint (169.88s)
--- PASS: TestAccS3BucketObjectsDataSource_encoded (171.78s)
--- PASS: TestAccS3BucketObjectsDataSource_fetchOwner (169.30s)
--- PASS: TestAccS3BucketObjectsDataSource_maxKeys (171.63s)
--- PASS: TestAccS3BucketObjectsDataSource_prefixes (173.14s)
--- PASS: TestAccS3BucketObjectsDataSource_startAfter (172.91s)

--- PASS: TestAccS3BucketOwnershipControls_Disappears_bucket (79.30s)
--- PASS: TestAccS3BucketOwnershipControls_Rule_objectOwnership (185.89s)
--- PASS: TestAccS3BucketOwnershipControls_basic (106.87s)
--- PASS: TestAccS3BucketOwnershipControls_disappears (90.36s)

--- PASS: TestAccS3BucketPolicy_IAMRoleOrder_jsonEncode (377.74s)
--- PASS: TestAccS3BucketPolicy_IAMRoleOrder_policyDoc (283.23s)
--- PASS: TestAccS3BucketPolicy_IAMRoleOrder_policyDocNotPrincipal (296.42s)

--- PASS: TestAccS3BucketPolicy_basic (105.43s)
--- PASS: TestAccS3BucketPolicy_policyUpdate (187.38s)

--- PASS: TestAccS3BucketPublicAccessBlock_Disappears_bucket (80.78s)
--- PASS: TestAccS3BucketPublicAccessBlock_basic (107.47s)
--- PASS: TestAccS3BucketPublicAccessBlock_blockPublicACLs (256.33s)
--- PASS: TestAccS3BucketPublicAccessBlock_blockPublicPolicy (251.33s)
--- PASS: TestAccS3BucketPublicAccessBlock_disappears (91.81s)
--- PASS: TestAccS3BucketPublicAccessBlock_ignorePublicACLs (252.53s)
--- PASS: TestAccS3BucketPublicAccessBlock_restrictPublicBuckets (249.35s)

--- PASS: TestAccS3BucketReplicationConfiguration_basic (546.66s)
--- PASS: TestAccS3BucketReplicationConfiguration_configurationRuleDestinationAccessControlTranslation (496.87s)
--- PASS: TestAccS3BucketReplicationConfiguration_configurationRuleDestinationAddAccessControlTranslation (484.87s)
--- PASS: TestAccS3BucketReplicationConfiguration_disappears (143.70s)
--- PASS: TestAccS3BucketReplicationConfiguration_filter_andOperator (362.20s)
--- PASS: TestAccS3BucketReplicationConfiguration_filter_tagFilter (364.50s)
--- PASS: TestAccS3BucketReplicationConfiguration_multipleDestinationsEmptyFilter (408.50s)
--- PASS: TestAccS3BucketReplicationConfiguration_multipleDestinationsNonEmptyFilter (406.62s)
--- PASS: TestAccS3BucketReplicationConfiguration_replicaModifications (398.14s)
--- PASS: TestAccS3BucketReplicationConfiguration_replicationTimeControl (399.96s)
--- PASS: TestAccS3BucketReplicationConfiguration_schemaV2 (394.49s)
--- PASS: TestAccS3BucketReplicationConfiguration_schemaV2DestinationMetrics (371.39s)
--- PASS: TestAccS3BucketReplicationConfiguration_schemaV2SameRegion (236.91s)
--- PASS: TestAccS3BucketReplicationConfiguration_twoDestination (408.63s)
--- PASS: TestAccS3BucketReplicationConfiguration_withoutPrefix (317.60s)
--- PASS: TestAccS3BucketReplicationConfiguration_withoutStorageClass (400.40s)

--- PASS: TestAccS3BucketVersioning_MFADelete (58.56s)
--- PASS: TestAccS3BucketVersioning_basic (97.08s)
--- PASS: TestAccS3BucketVersioning_disappears (87.84s)
--- PASS: TestAccS3BucketVersioning_update (145.05s)

--- PASS: TestAccS3Bucket_Basic_basic (62.94s)
--- PASS: TestAccS3Bucket_Basic_emptyString (62.95s)
--- PASS: TestAccS3Bucket_Basic_forceDestroy (96.49s)
--- PASS: TestAccS3Bucket_Basic_forceDestroyWithEmptyPrefixes (95.48s)
--- PASS: TestAccS3Bucket_Basic_forceDestroyWithObjectLockEnabled (96.41s)
--- PASS: TestAccS3Bucket_Basic_generatedName (35.14s)
--- PASS: TestAccS3Bucket_Basic_keyEnabled (66.70s)
--- PASS: TestAccS3Bucket_Basic_namePrefix (39.08s)
--- PASS: TestAccS3Bucket_Basic_requestPayer (62.93s)
--- PASS: TestAccS3Bucket_Basic_shouldFailNotFound (49.48s)
--- PASS: TestAccS3Bucket_Manage_MfaDeleteDisabled (69.01s)
--- PASS: TestAccS3Bucket_Manage_lifecycleBasic (232.74s)
--- PASS: TestAccS3Bucket_Manage_lifecycleExpireMarkerOnly (182.17s)
--- PASS: TestAccS3Bucket_Manage_lifecycleRuleAbortIncompleteMultipartUploadDaysNoExpiration (108.62s)
--- PASS: TestAccS3Bucket_Manage_lifecycleRuleExpirationEmptyBlock (91.56s)
--- PASS: TestAccS3Bucket_Manage_objectLock (151.91s)
--- PASS: TestAccS3Bucket_Manage_versioning (155.82s)
--- PASS: TestAccS3Bucket_Manage_versioningAndMfaDeleteDisabled (75.79s)
--- PASS: TestAccS3Bucket_Manage_versioningDisabled (68.90s)
--- PASS: TestAccS3Bucket_Replication_RTC_valid (256.45s)
--- PASS: TestAccS3Bucket_Replication_basic (307.43s)
--- PASS: TestAccS3Bucket_Replication_expectVersioningValidationError (85.43s)
--- PASS: TestAccS3Bucket_Replication_multipleDestinationsEmptyFilter (176.64s)
--- PASS: TestAccS3Bucket_Replication_multipleDestinationsNonEmptyFilter (172.64s)
--- PASS: TestAccS3Bucket_Replication_ruleDestinationAccessControlTranslation (230.81s)
--- PASS: TestAccS3Bucket_Replication_ruleDestinationAddAccessControlTranslation (220.84s)
--- PASS: TestAccS3Bucket_Replication_schemaV2 (298.67s)
--- PASS: TestAccS3Bucket_Replication_schemaV2SameRegion (115.22s)
--- PASS: TestAccS3Bucket_Replication_twoDestination (171.72s)
--- PASS: TestAccS3Bucket_Replication_withoutPrefix (160.59s)
--- PASS: TestAccS3Bucket_Replication_withoutStorageClass (166.47s)
--- PASS: TestAccS3Bucket_Security_aclToGrant (61.88s)
--- PASS: TestAccS3Bucket_Security_corsDelete (66.00s)
--- PASS: TestAccS3Bucket_Security_corsEmptyOrigin (87.10s)
--- PASS: TestAccS3Bucket_Security_corsUpdate (160.55s)
--- PASS: TestAccS3Bucket_Security_disableDefaultEncryptionWhenDefaultEncryptionIsEnabled (121.05s)
--- PASS: TestAccS3Bucket_Security_enableDefaultEncryptionWhenAES256IsUsed (58.93s)
--- PASS: TestAccS3Bucket_Security_enableDefaultEncryptionWhenTypical (56.78s)
--- PASS: TestAccS3Bucket_Security_grantToACL (77.11s)
--- PASS: TestAccS3Bucket_Security_logging (99.01s)
--- PASS: TestAccS3Bucket_Security_policy (95.44s)
--- PASS: TestAccS3Bucket_Security_updateACL (58.94s)
--- PASS: TestAccS3Bucket_Security_updateGrant (103.94s)
--- PASS: TestAccS3Bucket_Tags_basic (38.09s)
--- PASS: TestAccS3Bucket_Tags_ignoreTags (62.16s)
--- PASS: TestAccS3Bucket_Tags_withNoSystemTags (140.13s)
--- PASS: TestAccS3Bucket_Tags_withSystemTags (176.02s)
--- PASS: TestAccS3Bucket_Web_redirect (169.72s)
--- PASS: TestAccS3Bucket_Web_routingRules (105.39s)
--- PASS: TestAccS3Bucket_Web_simple (166.59s)

--- PASS: TestBucketName (0.00s)
--- PASS: TestBucketRegionalDomainName (0.00s)
--- PASS: TestWebsiteEndpoint (0.00s)
--- SKIP: TestAccS3BucketReplicationConfiguration_existingObjectReplication (0.00s)
```

### CODE REVIEW

**Comment by anGie44** [comment]
Optionally preceded by https://github.com/hashicorp/terraform-provider-aws/pull/22617

File: internal/service/s3/bucket_test.go [hunk 1]
```diff
@@ -360,43 +359,6 @@ func TestAccS3Bucket_Basic_generatedName(t *testing.T) {
 	})
 }
 
-func TestAccS3Bucket_Basic_acceleration(t *testing.T) {
```
Comment (anGie44) [inline internal/service/s3/bucket_test.go:363]: These tests have been ported over to #22617

**Review by YakDriver — APPROVED** [review]
Looks good! 🎉 

```console
% make testacc TESTS=TestAccS3Bucket PKG=s3
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./internal/service/s3/... -v -count 1 -parallel 20 -run='TestAccS3Bucket'  -timeout 180m
--- SKIP: TestAccS3BucketReplicationConfiguration_existingObjectReplication (0.00s)
--- PASS: TestAccS3Bucket_Security_corsDelete (93.46s)
--- PASS: TestAccS3Bucket_Basic_forceDestroy (100.02s)
--- PASS: TestAccS3Bucket_Basic_forceDestroyWithEmptyPrefixes (102.45s)
--- PASS: TestAccS3Bucket_Basic_forceDestroyWithObjectLockEnabled (108.61s)
--- PASS: TestAccS3Bucket_Security_corsEmptyOrigin (113.70s)
--- PASS: TestAccS3BucketVersioning_disappears (113.75s)
--- PASS: TestAccS3Bucket_Manage_versioningAndMfaDeleteDisabled (118.77s)
--- PASS: TestAccS3BucketAnalyticsConfiguration_basic (120.19s)
--- PASS: TestAccS3Bucket_Manage_MfaDeleteDisabled (123.52s)
--- PASS: TestAccS3Bucket_Basic_generatedName (126.33s)
--- PASS: TestAccS3BucketVersioning_basic (127.69s)
--- PASS: TestAccS3BucketVersioning_MFADelete (131.76s)
--- PASS: TestAccS3Bucket_Security_logging (137.68s)
--- PASS: TestAccS3Bucket_Replication_schemaV2SameRegion (144.28s)
--- PASS: TestAccS3Bucket_Replication_expectVersioningValidationError (69.87s)
--- PASS: TestAccS3Bucket_Basic_shouldFailNotFound (79.37s)
--- PASS: TestAccS3Bucket_Manage_objectLock (212.60s)
--- PASS: TestAccS3Bucket_Security_corsUpdate (225.93s)
--- PASS: TestAccS3Bucket_Security_enableDefaultEncryptionWhenAES256IsUsed (127.45s)
--- PASS: TestAccS3BucketOwnershipControls_basic (116.06s)
--- PASS: TestAccS3Bucket_Basic_keyEnabled (137.70s)
--- PASS: TestAccS3Bucket_Security_enableDefaultEncryptionWhenTypical (135.05s)
--- PASS: TestAccS3Bucket_Replication_withoutPrefix (190.93s)
--- PASS: TestAccS3Bucket_Replication_withoutStorageClass (183.38s)
--- PASS: TestAccS3Bucket_Basic_namePrefix (122.23s)
--- PASS: TestAccS3Bucket_Manage_lifecycleBasic (304.36s)
--- PASS: TestAccS3Bucket_Replication_ruleDestinationAccessControlTranslation (310.93s)
--- PASS: TestAccS3BucketVersioning_update (318.43s)
--- PASS: TestAccS3Bucket_Tags_basic (121.99s)
--- PASS: TestAccS3Bucket_Security_disableDefaultEncryptionWhenDefaultEncryptionIsEnabled (208.97s)
--- PASS: TestAccS3Bucket_Manage_versioning (221.78s)
--- PASS: TestAccS3Bucket_Web_routingRules (207.13s)
--- PASS: TestAccS3Bucket_Manage_lifecycleRuleExpirationEmptyBlock (76.20s)
--- PASS: TestAccS3BucketIntelligentTieringConfiguration_disappears (89.79s)
--- PASS: TestAccS3Bucket_Manage_lifecycleRuleAbortIncompleteMultipartUploadDaysNoExpiration (108.24s)
--- PASS: TestAccS3Bucket_Web_redirect (293.65s)
--- PASS: TestAccS3Bucket_Basic_emptyString (105.82s)
--- PASS: TestAccS3Bucket_Replication_twoDestination (170.73s)
--- PASS: TestAccS3Bucket_Tags_ignoreTags (216.67s)
--- PASS: TestAccS3Bucket_Replication_multipleDestinationsNonEmptyFilter (169.34s)
--- PASS: TestAccS3Bucket_Basic_basic (113.71s)
--- PASS: TestAccS3Bucket_Replication_multipleDestinationsEmptyFilter (154.10s)
--- PASS: TestAccS3Bucket_Replication_ruleDestinationAddAccessControlTranslation (336.02s)
--- PASS: TestAccS3BucketNotification_Topic_multiple (110.65s)
--- PASS: TestAccS3BucketNotification_topic (105.59s)
--- PASS: TestAccS3BucketOwnershipControls_Disappears_bucket (66.75s)
--- PASS: TestAccS3Bucket_Manage_versioningDisabled (101.78s)
--- PASS: TestAccS3BucketOwnershipControls_disappears (79.51s)
--- PASS: TestAccS3Bucket_Replication_RTC_valid (512.62s)
--- PASS: TestAccS3BucketAnalyticsConfiguration_WithStorageClassAnalysis_empty (3.80s)
--- PASS: TestAccS3BucketPolicy_basic (94.77s)
--- PASS: TestAccS3BucketNotification_update (214.14s)
--- PASS: TestAccS3BucketCorsConfiguration_SingleRule (72.68s)
--- PASS: TestAccS3Bucket_Security_aclToGrant (142.89s)
--- PASS: TestAccS3BucketAnalyticsConfiguration_WithFilter_empty (1.80s)
--- PASS: TestAccS3Bucket_Manage_lifecycleExpireMarkerOnly (148.50s)
--- PASS: TestAccS3BucketDataSource_basic (53.74s)
--- PASS: TestAccS3BucketAnalyticsConfiguration_WithFilter_singleTag (120.51s)
--- PASS: TestAccS3BucketOwnershipControls_Rule_objectOwnership (118.93s)
--- PASS: TestAccS3Bucket_Basic_requestPayer (145.53s)
--- PASS: TestAccS3Bucket_Tags_withNoSystemTags (315.59s)
--- PASS: TestAccS3BucketDataSource_website (53.67s)
--- PASS: TestAccS3BucketCorsConfiguration_disappears (44.27s)
--- PASS: TestAccS3BucketCorsConfiguration_basic (48.46s)
--- PASS: TestAccS3BucketAnalyticsConfiguration_WithFilter_prefixAndTags (105.24s)
--- PASS: TestAccS3Bucket_Tags_withSystemTags (342.02s)
--- PASS: TestAccS3Bucket_Security_updateACL (108.76s)
--- PASS: TestAccS3BucketAnalyticsConfiguration_WithFilter_multipleTags (91.59s)
--- PASS: TestAccS3BucketAnalyticsConfiguration_WithFilter_remove (100.52s)
--- PASS: TestAccS3BucketMetric_withEmptyFilter (2.24s)
--- PASS: TestAccS3Bucket_Security_policy (177.86s)
--- PASS: TestAccS3BucketAnalyticsConfiguration_WithStorageClassAnalysis_full (47.88s)
--- PASS: TestAccS3BucketIntelligentTieringConfiguration_basic (50.27s)
--- PASS: TestAccS3BucketCorsConfiguration_MultipleRules (47.49s)
--- PASS: TestAccS3Bucket_Replication_basic (413.14s)
--- PASS: TestAccS3BucketAnalyticsConfiguration_WithFilter_prefix (87.66s)
--- PASS: TestAccS3BucketAnalyticsConfiguration_WithStorageClassAnalysis_default (46.31s)
--- PASS: TestAccS3BucketAnalyticsConfiguration_removed (82.43s)
--- PASS: TestAccS3BucketMetric_basic (50.24s)
--- PASS: TestAccS3BucketInventory_encryptWithSSEKMS (50.32s)
--- PASS: TestAccS3BucketInventory_encryptWithSSES3 (49.46s)
--- PASS: TestAccS3BucketInventory_basic (50.84s)
--- PASS: TestAccS3BucketNotification_queue (91.86s)
--- PASS: TestAccS3BucketMetric_withFilterPrefix (82.24s)
--- PASS: TestAccS3Bucket_Replication_schemaV2 (599.67s)
--- PASS: TestAccS3BucketCorsConfiguration_update (125.09s)
--- PASS: TestAccS3BucketReplicationConfiguration_withoutPrefix (367.28s)
--- PASS: TestAccS3BucketAnalyticsConfiguration_updateBasic (133.75s)
--- PASS: TestAccS3BucketNotification_eventbridge (44.76s)
--- PASS: TestAccS3BucketNotification_LambdaFunctionLambdaFunctionARN_alias (65.98s)
--- PASS: TestAccS3BucketMetric_withFilterPrefixAndMultipleTags (80.64s)
--- PASS: TestAccS3BucketReplicationConfiguration_disappears (47.90s)
--- PASS: TestAccS3BucketMetric_withFilterSingleTag (64.91s)
--- PASS: TestAccS3BucketNotification_lambdaFunction (101.19s)
--- PASS: TestAccS3BucketMetric_withFilterMultipleTags (60.14s)
--- PASS: TestAccS3BucketReplicationConfiguration_filter_andOperator (439.52s)
--- PASS: TestAccS3BucketReplicationConfiguration_schemaV2SameRegion (178.54s)
--- PASS: TestAccS3BucketPublicAccessBlock_restrictPublicBuckets (89.56s)
--- PASS: TestAccS3BucketMetric_withFilterPrefixAndSingleTag (52.89s)
--- PASS: TestAccS3BucketIntelligentTieringConfiguration_Filter (146.41s)
--- PASS: TestAccS3BucketPublicAccessBlock_ignorePublicACLs (81.92s)
--- PASS: TestAccS3BucketPublicAccessBlock_disappears (31.56s)
--- PASS: TestAccS3BucketPublicAccessBlock_basic (32.16s)
--- PASS: TestAccS3BucketPublicAccessBlock_blockPublicACLs (81.27s)
--- PASS: TestAccS3BucketPolicy_IAMRoleOrder_policyDocNotPrincipal (112.59s)
--- PASS: TestAccS3BucketPublicAccessBlock_blockPublicPolicy (80.26s)
--- PASS: TestAccS3Bucket_Security_updateGrant (100.38s)
--- PASS: TestAccS3BucketReplicationConfiguration_schemaV2DestinationMetrics (305.04s)
--- PASS: TestAccS3BucketPolicy_policyUpdate (56.55s)
--- PASS: TestAccS3Bucket_Security_grantToACL (56.16s)
--- PASS: TestAccS3BucketPublicAccessBlock_Disappears_bucket (18.65s)
--- PASS: TestAccS3Bucket_Web_simple (93.43s)
--- PASS: TestAccS3BucketPolicy_IAMRoleOrder_policyDoc (95.10s)
--- PASS: TestAccS3BucketReplicationConfiguration_schemaV2 (296.47s)
--- PASS: TestAccS3BucketReplicationConfiguration_withoutStorageClass (298.06s)
--- PASS: TestAccS3BucketReplicationConfiguration_replicaModifications (295.08s)
--- PASS: TestAccS3BucketReplicationConfiguration_replicationTimeControl (313.09s)
--- PASS: TestAccS3BucketReplicationConfiguration_twoDestination (298.13s)
--- PASS: TestAccS3BucketReplicationConfiguration_multipleDestinationsNonEmptyFilter (286.67s)
--- PASS: TestAccS3BucketReplicationConfiguration_multipleDestinationsEmptyFilter (288.99s)
--- PASS: TestAccS3BucketReplicationConfiguration_configurationRuleDestinationAddAccessControlTranslation (333.42s)
--- PASS: TestAccS3BucketReplicationConfiguration_configurationRuleDestinationAccessControlTranslation (328.27s)
--- PASS: TestAccS3BucketReplicationConfiguration_basic (343.43s)
--- PASS: TestAccS3BucketPolicy_IAMRoleOrder_jsonEncode (684.10s)
--- PASS: TestAccS3BucketReplicationConfiguration_filter_tagFilter (275.57s)
PASS
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/s3	1151.594s
```

File: .changelog/22610.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+```release-note:note
```
Comment (anGie44) [inline .changelog/22610.txt:1]:
```suggestion
```release-note:breaking-change
```

**Comment by anGie44** [comment]
Output of updated acceptance tests:
```
--- PASS: TestAccS3BucketAccelerateConfiguration_basic (34.96s)
--- PASS: TestAccS3BucketAccelerateConfiguration_disappears (35.35s)
--- PASS: TestAccS3BucketAccelerateConfiguration_update (59.25s)
```

**Pull request merged**